addons/account/tests/test_chart_template.py

```diff
@@ -412,6 +412,16 @@ def local_get_data(self, template_code):
         self.assertEqual(len(children_taxes), 2, "Two children should have been created.")
         self.assertEqual(parent_tax.children_tax_ids.ids, children_taxes.ids, "The parent and its children taxes should be linked together.")
 
+        with patch.object(AccountChartTemplate, '_get_chart_template_data', side_effect=local_get_data, autospec=True):
```

Comment:
To be honest I don't understand why we can put the check above as we don't change anything?

Reply:
I'm not sure I understand your question.
What I want to show, is that even when reloading the taxes are not changed/recreated (while they were).

Before my fix, the parent_tax got her name changed, and a new tax with the name `Tax with children` was created.
Rechecking the number of taxes with the names will return the same values, as we recreate new ones. We would indeed have only one tax with the name `Tax with children` (we would have another one with the name `[old] Tax with children` which is what I'm checking)

(Maybe I could simply check that the name of the parent_tax is kept the same instead of doing a new search/search_count btw :smile: )

Reply:
Forgot what I said :grimacing: